python/pyspark/ml/wrapper.py

```diff
@@ -190,8 +190,30 @@ def _transform(self, dataset):
         return DataFrame(self._java_obj.transform(dataset._jdf), dataset.sql_ctx)
 
 
+class JavaCallable(object):
```

Comment:
Ok, cool.  So is it good enough as-is for this PR, and then I can open another JIRA for the changes mentioned above?